src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -16,7 +14,7 @@ services:
   db:
     # on the verge of every deployment (fortnightly), update the "image" value from the "db" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json
-    image: daschswiss/apache-jena-fuseki:2.1.1
+    image: daschswiss/apache-jena-fuseki:2.1.2
```

Comment:
@seakayone Normally we use the versions defined in https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json. Is there a problem with apache-jena-fuseki 2.1.1? Can you give me some context?

Reply:
The purpose is to upgrade the JVM 17. I have already created PR to update api and deploy as well. 
See https://linear.app/dasch/issue/DEV-2743/update-jena-fuseki-container-to-java-17 for more details.

Reply:
Ah okay, I see. If you agree, I put this PR on hold. 

We only bump versions according to what is on prod. That means we wait until 1 day before deployment to prod, and then we bump the versions in this file according to exactly what stands in https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json. 

Since tomorrow's deployment is postponed, we have to wait anyways.

But thank you very much for remembering us 😄